plugin/evm/validators_cache_test.go

```diff
@@ -100,13 +99,11 @@ func TestValidatorsCache_ByEpoch(t *testing.T) {
 		assert.NoError(t, err)
 		assert.Empty(t, got)
 		assert.ElementsMatch(t, testValidators[epoch], got)
```

Comment:
How is `assert.ElementsMatch` different from `assert.Equal` in the case?

Reply:
assert.ElementsMatch works successfully with lists - arrays or slices, not maps.

Reply:
You mean assert.Equal in the above sentence?

Reply:
No, 
I meant `assert.ElementsMatch`.
About the `assert.Equal` fails randomly when comparing maps,
and I prefer to test the maps either using reflect.DeepEqual() or inspect their keys and length myself.
Here I used the second one.